src/Main.java

```diff
@@ -0,0 +1,29 @@
+import java.util.List;
```

Comment:
You should use  a .gitignore file to avoid checking in unnecessary files like .idea/, *.iml, target/, etc.